app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -434,6 +434,12 @@ export default Ember.Component.extend({
           const full = `:${term}`;
           term = term.toLowerCase();
 
+          if (term.length < 1 && I18n.locale == "fr") {
```

Comment:
Instead of testing for the French locale I would add a new site setting that controls the minimum number of characters before showing up the emoji modal.

Also we use `===` to test for equality ;)